micrometer-spring-legacy/src/test/java/io/micrometer/spring/web/servlet/MetricsFilterTest.java

```diff
@@ -94,7 +94,7 @@ public void timedMethod() throws Exception {
 
         assertThat(this.registry.find("http.server.requests")
             .tags("status", "200", "uri", "/api/c1/{id}", "public", "true")
-            .value(Statistic.Count, 1.0).timer()).isPresent();
+            .timer().count()).isEqualTo(1L);
```

Comment:
This test it the only one I updated to the new API. Figured I would get your approval @jkschneider  before touching all those tests one more time.